locales/en.json

```diff
@@ -71,28 +71,28 @@
 	"Header": "Header",
 	"Unordered List": "Unordered List",
 	"Ordered List": "Ordered List",
-	"Todo List": "Todo List",
+	"Todo List": "Checklist",
```

Comment:
Not all of these will be todo lists. "Task List" is definitely better than "Todo List", but personally I'd prefer "checklist".